cmd/neofs-adm/internal/modules/morph/local_client.go

```diff
@@ -151,6 +153,36 @@ func (l *localClient) CreateTxFromScript(script []byte, acc *wallet.Account, sys
 	return tx, nil
 }
 
+func (l *localClient) GetCommittee() (keys.PublicKeys, error) {
+	//TODO implement me
```

Comment:
We can remove this TODO and replace it with something like `not used by morph init`, because it is the only place where local client is intended to be used.

Reply:
yeah, that was added autimatically by my IDE, wanted to see your suggestions and fix that. changed the comment, panic is kept